### ISSUE TITLE
Tower completion rewards

### ISSUE THREAD

**Issue by TheFoolAsWell** [fuzzything44/Incremental#16]
Thoughts about what rewards should be given for completing the tower.

**Comment by TheFoolAsWell** [comment]
1. Reduce time for small tower refresh
   1. Halve the time (to 11.5hrs)
   2. Halve the time again (to 5hr 45 minutes).
   3. Halve yet again (to 2hr 52.5)
   4. Reduce to 1hr. 

2. Additional spells
   1. Convert mana to essence at normal rate.  Costs 100 mana to use.  1/sec.
   2. Workshop like spell.  Converts mana to essence at reduced rate.  Costs 1000 mana to use.  Converts 10/30 seconds at half cost rate.  Call it "Essence lab" possibly.
   3. Improvement on (2).  Converts 100/120 seconds at 1/4 cost rate.
   4. Improvement on (2).  Converts 1000/300 seconds at 1/8 cost rate.


3. Improvements to tavern and tower of magic:
   1. Power gain x5, toughness gain x4 in tower of magic.
   2. Warrior gain x10 (both power and toughness).
   3. Healer gain x10

Suggested order of rewards:

1. 1.i
2. 2.i
3. 1.ii
4. 3.i
5. 1.iii
6. 2.ii
7. 3.ii
8. 2.iii
9. 3.iii (or possibly 3.i again)
10. 1.iv
11. 2.iv

Some of the rewards may well need balancing.  In particular, speed of Essence lab may need changing (speeding up?) and multipliers to power and toughness gains.

**Comment by fuzzything44** [comment]
I'd prefer to keep all Mana -> Essence conversions covered by one building (also, it probably won't need upgrades for a long time because it's so mana hungry, so having those as new floors feels kind of bad). 

I'd say make Ascension I rewards something like:
1. Reduce tower time to 11hr (halving + apply hour reduction after)
2. New Tavern Drinks (that give a better rate and are heavily biased to one side)
3. New Tavern Drinks (same, but for other side)
4. Mana -> Essence spell

Ascension II can be worked out later (also, it'll be pretty hard to get to considering the difficulty increase)

**Comment by TheFoolAsWell** [comment]
> Ascension II can be worked out later (also, it'll be pretty hard to get to considering the difficulty increase)

I think that was part of my reasoning for improving mana -> essence rate.  I.e. there needs to be a way of producing more essence from mana.  However, you may well be right.  Putting all the types into one ascension does speed up the help, and may mean that reductions more easily.

More specific comments:

1. Halving then applying hour reduction afterwards makes more sense since you lose the hour reduction to start with.

2. and 3.  Do you mean for them both to be given at the same time?  I like the idea of new drinks, but would it make sense to think in terms of "rounds".  I.e. you spend some much essence and get improvements to everyone --- not necessarily equally.  then a different "round" could give a different balance of improvements.

4. If you feel that this wants to be a single spell that's fine.  It makes things simpler.

**Comment by TheFoolAsWell** [comment]
Other thoughts:  

1.  Leave essence cost reset on small tower.  That way more essence can be earned and give a better chance of being able to get to later ascensions.

2.  Ascension II would, among other things, grant a second halving to small tower time (to 5 effectively) and then ascension III again (to 2).  That would probably be enough.  Is this too powerful?

**Comment by fuzzything44** [comment]
No, not both given at the same time. Ascension I floor 2 gives a new drink that increases power or toughness at a better rate than normal (for both), and floor 3 gives a new drink that increases the one that 2 doesn't increase.

Small tower should keep the essence cost reset, yes. 
My concern with small tower time getting too short is it forces too active of play - potentially add a multiplier to gains from it instead (or have you get X "entries" / day)

**Comment by fuzzything44** [comment]
Okay, Ascension I is done (oh, also the official game uses Roman Numerals! Neat!). II can probably wait a bit, but not long as I is same difficulty multiplier as 0. II will be 2x multiplier.

As for Ascension II rewards, I'm thinking something to improve mana, some changeable party member (hire one of a few at the Tavern) (probably final reward), and then I'm not quite sure. Maybe add banked tower of grinding entries (so you can enter every 6 hours or something, but it also stacks up to 4 entries). That's 3, so then a 4th and we're set.

Ascension III should give another option for the changeable party member, probably an upgrade to Essence Compressor, and then I'm not sure what.

**Comment by TheFoolAsWell** [comment]
I think there ought to be more drinks, although the current drinks need to be downgraded.  It is too easy to get 3-5k of power for warrior or healer.  Either or both overpower. 
Perhaps the further drinks can wait until Ascension III.

**Comment by fuzzything44** [comment]
Ah wow. I might have messed up in costing them then...

**Comment by TheFoolAsWell** [comment]
I think the face punch should be reduced to at most 2 and 3 respectively for healer and warrior (power gain).  Probably 1 and 5/3.  Equally, warrior iron juice probably wants to drop toughness amount (to 10?).  Not sure.  Didn't even consider using lots given the other options.
